pom.xml

```diff
@@ -193,15 +193,17 @@
             org.codehaus.groovy.*;resolution:=optional,\
             kotlin.*;resolution:=optional,\
             io.vertx.groovy.*;resolution:=optional,\
-            io.vertx.ext.auth,\
-            io.vertx.ext.auth.*;resolution:=optional,\
```

Comment:
Why are these removed and not io.vertx.rxjava.ext.auth ?

Reply:
I forgot them :)
thanks for the head-up